File: app/src/test/java/pemwa/com/javadevsnairobi/GithubUsersTest.java
package pemwa.com.javadevsnairobi;

import org.junit.Test;

import pemwa.com.javadevsnairobi.model.GithubUsers;

import static org.junit.Assert.assertEquals;

public class GithubUsersTest {

    private String username = "username";
    private String userProfile = "userProfile";
    private String url = "url";
    private String followers = "followers";
    private String repos = "repos";

    private GithubUsers githubUsers = new GithubUsers(username, userProfile, url, followers, repos);

    @Test
    public void testUserData() {
        assertEquals(username, githubUsers.getUserName());
        assertEquals(userProfile, githubUsers.getProfilePic());
        assertEquals(url, githubUsers.getUrl());
        assertEquals(followers, githubUsers.getFollowers());
        assertEquals(repos, githubUsers.getRepos());
    }
}

File: app/src/test/java/pemwa/com/javadevsnairobi/GithubUsersTest.kt
package pemwa.com.javadevsnairobi

import org.junit.Test

import pemwa.com.javadevsnairobi.model.GithubUsers

import org.junit.Assert.assertEquals

class GithubUsersTest {

    private val username = "username"
    private val userProfile = "userProfile"
    private val url = "url"
    private val followers = "followers"
    private val repos = "repos"

    private val githubUsers = GithubUsers(username, userProfile, url, followers, repos)

    @Test
    fun testUserData() {
        assertEquals(username, githubUsers.userName)
        assertEquals(userProfile, githubUsers.profilePic)
        assertEquals(url, githubUsers.url)
        assertEquals(followers, githubUsers.followers)
        assertEquals(repos, githubUsers.repos)
    }
}
